app/assets/javascripts/map.js

```diff
@@ -4,16 +4,26 @@ $(function() {
 
   var createMap = function(latitude, longitude){
     L.mapbox.accessToken = window.MAPBOX_ACCESS_TOKEN;
-    map = L.mapbox.map('map', window.MAPBOX_USER_NAME).
+    map = L.mapbox.map('map', window.MAPBOX_USER_NAME, { zoomControl: false}).
       setView([latitude, longitude], 14);
   };
 
   var createMarker = function(court) {
     var marker = L.marker([ court.latitude, court.longitude], {
       icon: L.mapbox.marker.icon({
-        'marker-color': '#9c89cc'
+        'marker-color': '#9c89cc',
+        'clickable': true
       })
-    }).addTo(map);
+    })
+    .addTo(map);
+    marker.on("click", function(){
```

Comment:
I'm not sure if I could do what I want otherwise. Take a look at the description